source/tutorials/write-update-documents.txt

```diff
@@ -101,42 +94,50 @@ To learn more about update operators, see
 Update One Document
 -------------------
 
-To update a single document from a MongoDB collection, call the ``updateOne()``
-method and pass your query filter as the first argument. Then, pass your
-update operators as the second argument to the ``updateOne()`` method. After you call the
-``updateOne()`` method, chain the ``subscribe()`` method and pass a subscriber
-as an argument.
-
-The following example uses ``Updates.set()`` to update the ``name``
+To update a single document in a MongoDB collection, call the ``updateOne()``
+method and pass your query filter and update operators. Then, pass the
+``updateOne()`` result to the static ``Mono.from()`` method from
+``Mono``. ``Mono`` is a class from the Project Reactor library. In {+java-rs+}, 
+the driver methods return cold ``Publishers``, which means that the
+corresponding operation does not happen unless you subscribe to the returned
+``Publisher``. This guide uses the Project Reactor library to consume them.To learn more
+about ``Mono``, see `Mono <https://projectreactor.io/docs/core/release/reference/#mono>`__ in the Project
+Reactor documentation.
+
+The following example uses the ``updateOne()`` method to update the ``name``
 value of a matching document from ``"Bagels N Buns"`` to ``"2 Bagels 2 Buns"``:
 
 .. code-block:: java
    :copyable: true
 
-    ObservableSubscriber<UpdateResult> updateOneSubscriber = new SubscriberHelpers.OperationSubscriber<>();
-    restaurants.updateOne(eq("name", "Bagels N Buns"), set("name", "2
-    Bagels 2 Buns")).subscribe(updateOneSubscriber);
-    updateOneSubscriber.await();
+    Publisher<UpdateResult> updatePublisher = 
+            restaurants.updateOne(eq("name", "Bagels N Buns"),
+                    set("name", "2 Bagels 2 Buns"));
+    Mono.from(updatePublisher).block();
 
 Update Multiple Documents
 -------------------------
 
-To update multiple documents from a MongoDB collection, call the ``updateMany()``
-method and pass your query filter. Then, pass your
-update operators as the second argument to the ``updateMany()`` method. After you call the
-``updateMany()`` method, chain the ``subscribe()`` method and pass a subscriber
-as an argument.
+To update multiple documents in a MongoDB collection, call the ``updateMany()``
+method and pass your query filter and update operators. Then, pass the ``updateMany()`` result to the static ``Mono.from()`` method from
+``Mono``. ``Mono`` is a class from the Project Reactor library. In {+java-rs+}, 
+the driver methods return cold ``Publishers``, which means that the
+corresponding operation does not happen unless you subscribe to the returned
+``Publisher``. This guide uses the Project Reactor library to consume them.To learn more
+about ``Mono``, see `Mono <https://projectreactor.io/docs/core/release/reference/#mono>`__ in the Project
+Reactor documentation.
```

Comment:
[optional, feel free to ignore]

This note on `Mono` and `Publisher` is a verbatim copy of the note from the "Update One Document" section on this same page. I am not sure it's a good idea to duplicate such notes, as it requires a reader to either re-read it.

Reply:
The reason we verbatim copy the note is because people can easily jump to just the section they want (ex: update one, or update multiple) and we cannot assume that people have read the previous sections.